Gruntfile.js

```diff
@@ -373,11 +421,40 @@ module.exports = function (grunt) {
             "clean",
             "lint",
             "distributions" + ( target ? ":" + target : "" ),
-            "cleanForDist"
+            "cleanForDist",
+            "verifyDists"
         ];
         grunt.task.run(tasks);
     });
 
+    grunt.registerTask("verifyDists", "Verifies that the expected /dist/*.js files and their source maps were created", function () {
+        console.log("Verifying that expected distribution files are present in /dist directory");
```

Comment:
Similar for below, and you may want to use the various types of logs they have available.